.github/workflows/e2e_run_all.yaml

```diff
@@ -0,0 +1,179 @@
+# (C) Copyright Confidential Containers Contributors 2023.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Run end-to-end (e2e) tests.
+---
+name: (Callable) Run all e2e tests
+
+on:
+  workflow_call:
+    inputs:
+      authorized:
+        default: false
+        description: Pre-authorize the workflow execution
+        required: false
+        type: boolean
+      caa_image_tag:
+        description: set the cloud-api-adaptor image tag
+        required: true
+        type: string
+      git_ref:
+        default: 'main'
+        description: Git ref to checkout the cloud-api-adaptor repository. Defaults to main.
+        required: false
+        type: string
+      podvm_image_tag:
+        description: set the podvm_builder/podvm_binaries/podvm image tag
+        required: true
+        type: string
+      registry:
+        description: the container registry where built images will be pushed to
+        required: true
+        type: string
+env:
+  # cloud-api-adaptor image registry
+  E2E_IMG_REGISTRY: ${{ inputs.registry }}
+  # cloud-api-adaptor: image release tag
+  E2E_IMG_RELEASE_TAG: ${{ inputs.caa_image_tag }}
+  # cloud-api-adaptor image dev tag
+  E2E_IMG_DEV_TAG: ${{ inputs.caa_image_tag }}-dev
+
+jobs:
+  authorize:
+    runs-on: ubuntu-latest
+    if: |
+      inputs.authorized == true ||
+      contains(github.event.pull_request.labels.*.name, 'test_e2e_libvirt')
```

Comment:
previously this condition was there and now you added the `inputs.authorized`, are you sure you want to use this to force-enable it rather than only check when authorized is set? I think `&&` would fit better here (but I may be wrong)